deployment-e2e/molecule.sh

```diff
@@ -2,8 +2,10 @@
 cd $(dirname $0)
 set -e # exit when any command fails
 
+CODEBASE_BRANCH=${CIRCLE_BRANCH-$(git symbolic-ref --short HEAD)}
```

Comment:
will it still be possible to run `e2e` deployment tests locally?

Reply:
Good question.
- In a normal repository - yes
- In a forked repository - I will work on it now and make sure that it is possible